src/main/java/adris/altoclef/tasks/container/LootContainerTask.java

```diff
@@ -57,34 +61,49 @@ protected Task onTick(AltoClef mod) {
         }
         Optional<Slot> optimal = getAMatchingSlot(mod);
         if (optimal.isEmpty()) {
+            _weDoneHere = true;
             return null;
         }
-        setDebugState("Looting a container for all of their " + target.toString());
+        setDebugState("Looting items: " + targets);
         return new ClickSlotTask(optimal.get());
     }
 
     @Override
     protected void onStop(AltoClef mod, Task task) {
-        if (!_wasProtected) {
-            mod.getBehaviour().removeProtectedItems(target);
+        if (ContainerType.screenHandlerMatches(ContainerType.CHEST)) StorageHelper.closeScreen();
+        for (Item item : _protected) {
+            mod.getBehaviour().removeProtectedItems(item);
```

Comment:
This is not necessary because `mod.getBehaviour().pop()` deletes the array

Reply:
Oh well